src/main/java/com/example/petbuddybackend/dto/offer/ModifyConfigurationDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.example.petbuddybackend.dto.offer;
+
+import lombok.Builder;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+@Builder
+public record ModifyConfigurationDTO(
+        String description,
+        BigDecimal dailyPrice,
```

Comment:
W entity mamy nullable = false, to może by tu dodać not null i minimalną wartość 0?